others/RESOURCES.md

```diff
@@ -11,3 +11,13 @@ Anything useful or informative that can be found on the internet, including arti
 * [mCoding](https://www.youtube.com/@mCoding) - Python
 * [Luke Smith](https://www.youtube.com/@LukeSmithxyz) - Linux, Shells, Life
 * [Hussein Nassar](https://www.youtube.com/@hnasr) - Backend System Design
+* [Honeypot](https://www.youtube.com/@Honeypotio) - Documentaries about different technologies, Carrer  and Life advice by different developers, Interviews with creators of famous tools and technologies (Thomas Dohmke - GitHub CEO, Bjarne Stroustrup - Creator of C++)
```

Comment:
Career is written incorrectly, double whitespace in-between Career and life. Life should be lowercase.

Reply:
Fixed